core/geometry/src/serialization/BGFBAccessors.ts

```diff
@@ -13,6120 +14,6396 @@ import { flatbuffers } from "flatbuffers";
  * @internal
  */
 export namespace BGFBAccessors {
-// automatically generated by the FlatBuffers compiler, do not modify
-
-/**
- * @enum {number}
- */
-export enum LoopTypeEnum {
-  Parity = 1,
-  InteriorToLeft = 2
-}
-
-/**
- * @enum {number}
- */
-export enum VariantGeometryUnion {
-      tagNONE = 0,
-      tagLineSegment = 1,
-      tagEllipticArc = 2,
-      tagBsplineCurve = 3,
-      tagLineString = 4,
-      tagCurveVector = 5,
-      tagDgnCone = 6,
-      tagDgnSphere = 7,
-      tagDgnTorusPipe = 8,
-      tagDgnBox = 9,
-      tagDgnExtrusion = 10,
-      tagDgnRotationalSweep = 11,
-      tagDgnRuledSweep = 12,
-      tagPolyface = 13,
-      tagBsplineSurface = 14,
-      tagVectorOfVariantGeometry = 15,
-      tagInterpolationCurve = 16,
-      tagTransitionSpiral = 17,
-      tagPointString = 18,
-      tagAkimaCurve = 19,
-      tagCatenaryCurve = 20,
-      tagPartialCurve = 21
-}
-
-/**
- * @constructor
- */
-export class DPoint3d {
-  bb: flatbuffers.ByteBuffer | null = null;
-
-  bb_pos: number = 0;
-/**
- * @param number i
- * @param flatbuffers.ByteBuffer bb
- * @returns DPoint3d
- */
-__init(i: number, bb: flatbuffers.ByteBuffer): DPoint3d {
-  this.bb_pos = i;
-  this.bb = bb;
-  return this;
-}
-
-/**
- * @returns number
- */
-x(): number {
-  return this.bb!.readFloat64(this.bb_pos);
```

Comment:
// automatically generated by the FlatBuffers compiler, do not modify
Do not change this file, just disable the warning.